Cloudinary.Test/UploadMappingApiTest.cs

```diff
@@ -51,13 +48,33 @@ public void TestUploadMapping()
             result = m_cloudinary.UploadMappings(new UploadMappingParams());
             Assert.IsFalse(result.Mappings.ContainsKey(FOLDERS[0]));
             Assert.IsFalse(result.Mappings.ContainsValue(NEW_TEMPLATE));
+            delMappings(FOLDERS);
+
+
+        }
+        private void delMappings(string[] folders)
```

Comment:
This method should be placed in the "private" region(which can be in the beginning or end of the file)
- in other words let's make all tests method in a region and have helper methods in a designated region.